treeherder/etl/perf.py

```diff
@@ -128,9 +128,13 @@ def _test_should_gather_replicates_based_on(
     mozilla-central.
     """
     if replicates and len(replicates) > 0:
-        return repository.name in ("try",) or (
-            repository.name in ("mozilla-central",) and suite_name == "speedometer3"
-        )
+        if repository.name in ("try",):
+            return True
+        elif repository.name in ("mozilla-central",):
+            if suite_name == "speedometer3":
+                return True
+            elif "applink-startup" in suite_name:
+                return True
```

Comment:
nit: What do you think of doing just one check like before 

1. `if (suite_name == "speedometer3") or ("applink-startup" in suite_name)`

2. 
```
return repository.name in ("try",) or 
(repository.name in ("mozilla-central",) and ((suite_name == "speedometer3") or ("applink-startup" in suite_name)))
```

Reply:
I think it was fine to have the one-liner before since it was just one test. However, I have a feeling we'll be adding more tests in the near future so it'll make it harder to read the code if we keep using the one-line setup.